search/query/date_range.go

```diff
@@ -144,15 +151,15 @@ func (q *DateRangeQuery) parseEndpoints() (*float64, *float64, error) {
 	max := math.Inf(1)
 	if !q.Start.IsZero() {
 		startInt64 := q.Start.UnixNano()
-		if startInt64 < 0 {
+		if startInt64 < 0 && q.Start.After(epochTime) {
```

Comment:
I think, this check only takes care of the end limit of year 2262, not the starting range year of 1678? What if any of the range values goes beyond that?

Reply:
Good point.